src/Sulu/Bundle/WebsiteBundle/EventListener/RedirectExceptionSubscriber.php

```diff
@@ -75,9 +75,16 @@ public static function getSubscribedEvents()
     /**
      * Redirect trailing slashes or ".html".
      */
-    public function redirectTrailingSlashOrHtml(GetResponseForExceptionEvent $event)
+    public function redirectTrailingSlashOrHtml(ExceptionEvent $event)
     {
-        if (!$event->getException() instanceof NotFoundHttpException) {
+        if (method_exists($event, 'getThrowable')) {
+            $exception = $event->getThrowable();
+        } else {
+            // remove this when symfony 4.3 support is dropped
```

Comment:
Didn't we decide that we drop Symfony 4.3 support?

Reply:
If we merge this one I could rebase #5238 where I remove this lines and you can then have a look at that one or we merge #5238 before merging this.